src/liana/plotting/_plot_spatial_inflow_score_single_interaction.py

```diff
@@ -0,0 +1,418 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+from matplotlib.gridspec import GridSpec
+import warnings
+from typing import List, Tuple, Optional, Union
+
+def plot_spatial_inflow_score_single_interaction(
```

Comment:
Lets try to think of a snappier name, this is a bit excessive :) I would also not limit it to inflow scores; you are plotting multiple features across cell types on a slide; `spatial_features_by_obs` or `spatial_features_by_group`